### DISCUSSION TITLE
为什么clash里只有两个，需要设置什么吗

### DISCUSSION THREAD

**Discussion by bbuus** [vvbbnn00/WARP-Clash-API#139] [Q&A] [answered]
<img width="646" alt="屏幕截图 2024-03-10 211555" src="https://github.com/vvbbnn00/WARP-Clash-API/assets/105034417/b2041cce-f3fc-40bb-b438-bb9e84730ae9">

**Comment by bbuus** [comment, marked as answer]
而且无法使用，望解答

**Reply by vvbbnn00** [reply]
需要添加订阅

**Reply by bbuus** [reply]
是的，我已经在20010中复制了订阅链接，但好像还是只有两个

**Reply by bbuus** [reply]
已经输入链接，然后获取订阅地址，接着在clash里订阅。但好像没有效果

**Reply by vvbbnn00** [reply]
检查一下clash的内核是不是不支持wireguard（例如clash for windows的默认内核）

如果是内核问题的话，可以更换到meta内核，或者使用目前仍在维护的 clash verge rev

**Reply by bbuus** [reply]
我已经下载了这个，并且内核就是meta

**Reply by vvbbnn00** [reply]
看界面好像有点旧啊，下载好配置以后点击订阅文件没有弹出错误提示之类的吗？

**Reply by bbuus** [reply]
我重新将项目导入docker试试。请稍等

**Reply by bbuus** [reply]
还是不行

**Reply by vvbbnn00** [reply]
这张截图无法定位问题，可以尝试更换 clash verge rev 试试，若问题仍然存在，麻烦将整个操作流程截屏一下

**Reply by bbuus** [reply]
已经可以了，谢谢大佬！

**Reply by bbuus** [reply]
![IMG_20240311_030523.jpg](https://github.com/vvbbnn00/WARP-Clash-API/assets/105034417/c1b7821a-0c59-4000-96f7-8a37afc149b1)

大佬，又有新问题了，出来了，但是不能用

**Reply by bbuus** [reply]
![IMG_20240311_030523.jpg](https://github.com/vvbbnn00/WARP-Clash-API/assets/105034417/2ec09436-e544-4e83-86ef-46f72101b748)

**Comment by bbuus** [comment]
<img width="846" alt="屏幕截图 2024-03-11 023737" src="https://github.com/vvbbnn00/WARP-Clash-API/assets/105034417/a02fbf4d-e56a-4981-99c9-d8573579fba0">
<img width="811" alt="屏幕截图 2024-03-11 023759" src="https://github.com/vvbbnn00/WARP-Clash-API/assets/105034417/c1941a49-6949-48f4-a956-049ac9207d39">

**Comment by bbuus** [comment]
运行和优选IP时显示这个

**Comment by bbuus** [comment]
运行和优选IP时显示这个

**Reply by pillarcoin** [reply]
建议你先去看一遍clash的wiki
